futurex_openedx_extensions/dashboard/serializers.py

```diff
@@ -506,6 +511,15 @@ def get_tenants(self, obj: get_user_model) -> Any:
         """Return the tenants."""
         return self.roles_data.get(obj.id, {}) if self.roles_data else {}
 
+    def get_global_roles(self, obj: get_user_model) -> Any:  # pylint:disable=no-self-use
+        """Return the global roles."""
+        global_roles = []
+        roles_dict = get_user_course_access_roles(obj)['roles']
+        for role, details in roles_dict.items():
+            if details.get('global_role', False):
+                global_roles.append(role)
+        return global_roles
```

Comment:
this works great @tehreem-sadat , thank you!

now, I'll drift you away from this for-loop. I added the existing `global_role` flag before realizing that global roles can only be global. I mean, we can know if the role is global or not by its name!

Can you please remove `'global_role'` completely from `get_user_course_access_roles` and refactor `get_global_roles` accordingly?

Reply:
@shadinaif Changes have been made PR is ready for re-review.

Reply:
thank you!